README.md

```diff
@@ -1,36 +1,70 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15555388&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
 
+software engineering is the systematic application of engineering principles, methods and tools to the development and maintenance of high-quality software principles.
+software engineering helps in the creating of software applications and systems that power various aspects of modern life such as transport and logistics, healthcare, banking and finance and etc.
+
 
 Identify and describe at least three key milestones in the evolution of software engineering.
```

Comment:
Phases of software development life cycle includes;
  - Requirements: Gathering and documenting user needs and system requirements.
  - Design: Creating high-level and detailed designs of the software architecture and user interface.
  - Implementation: Writing code and building the software according to the design specifications.
  - Testing: Conducting various tests to ensure the software meets quality standards and functional requirements.
  - Deployment: Releasing the software to users or customers.
  - Maintenance: Providing ongoing support, updates, and enhancements to the software after deployment.

Reply:
Phases of software development cycle
1.Planning
we define the software purpose and scope.

2.Requirement Analysis
identify and record the precise requirement of final users.

3.Design
This is about building the framework.

4.Coding
This is the actual process of writing code.

5.Testing
Check for errors and bugs by making sure the system does everything it's intended to.

6.Deployment 
Rolling out the fine tuned application to users.

7.Maintenance
Ensuring that the system is constantly improved .

Reply:
1.Planning: the initial phase involves defining the project’s goals, scope, and feasibility.

2.Requirements Analysis: In this phase, detailed requirements are gathered and documented. 

3.Design: During the design phase, the software architecture and user interfaces are planned.

4.Development (or Implementation): This phase is where the actual coding takes place.

5.Testing: After development, the software undergoes rigorous testing to identify and fix any defects.

6.Deployment: Once testing is complete and the software is deemed ready, it is deployed to the production environment. 

7.Maintenance: This phase involves ongoing support and updates to the software.

Reply:
Phases of the Software Development Life Cycle (SDLC) include the following:

Planning: Specify the goals, parameters, and viability of the project.

Requirement analysis: Compile and record the system and user requirements.

Design: Construct the system's architecture and comprehensive design.

Implementation: Write the actual code and work on it.

Testing: Make sure the software is operating appropriately by validating and verifying it.

Deployment: Make the program available to customers or users.

Maintenance: Apply updates, correct bugs, and offer continuing assistance.